packages/stargate/src/queries/ibc.spec.ts

```diff
@@ -14,220 +14,562 @@ async function makeClientWithIbc(rpcUrl: string): Promise<[QueryClient & IbcExte
+          tmClient.disconnect();
+        });
       });
-    });
 
-    describe("connections", () => {
-      it("works", async () => {
-        pendingWithoutSimapp();
-        const [client, tmClient] = await makeClientWithIbc(simapp.tendermintUrl);
+      describe("consensusStates", () => {
+        it("works", async () => {
+          pendingWithoutSimapp();
+          const [client, tmClient] = await makeClientWithIbc(simapp.tendermintUrl);
+
+          const response = await client.ibc.unverified.client.consensusStates(ibcTest.clientId);
+          expect(response.consensusStates).toEqual(
+            jasmine.arrayContaining([
+              {
+                height: jasmine.anything(),
+                consensusState: {
+                  typeUrl: "/ibc.lightclients.tendermint.v1.ConsensusState",
+                  value: jasmine.any(Uint8Array),
+                },
+              },
+            ]),
+          );
+
+          tmClient.disconnect();
+        });
+      });
 
-        const response = await client.ibc.unverified.connections();
-        expect(response.connections).toEqual([ibcTest.identifiedConnection]);
-        expect(response.pagination).toBeDefined();
-        expect(response.pagination).not.toBeNull();
-        expect(response.height).toBeDefined();
-        expect(response.height).not.toBeNull();
+      describe("allConsensusStates", () => {
+        it("works", async () => {
+          pendingWithoutSimapp();
+          const [client, tmClient] = await makeClientWithIbc(simapp.tendermintUrl);
+
+          const response = await client.ibc.unverified.client.allConsensusStates(ibcTest.clientId);
+          expect(response.consensusStates).toEqual(
+            jasmine.arrayContaining([
+              {
+                height: jasmine.anything(),
+                consensusState: {
+                  typeUrl: "/ibc.lightclients.tendermint.v1.ConsensusState",
+                  value: jasmine.any(Uint8Array),
+                },
+              },
+            ]),
+          );
+
+          tmClient.disconnect();
+        });
+      });
 
-        tmClient.disconnect();
+      describe("params", () => {
+        it("works", async () => {
+          pendingWithoutSimapp();
+          const [client, tmClient] = await makeClientWithIbc(simapp.tendermintUrl);
+
+          const response = await client.ibc.unverified.client.params();
+          expect(response.params).toEqual({
+            allowedClients: ["06-solomachine", "07-tendermint"],
+          });
+
+          tmClient.disconnect();
+        });
+      });
+
+      describe("stateTm", () => {
+        it("works", async () => {
+          pendingWithoutSimapp();
+          const [client, tmClient] = await makeClientWithIbc(simapp.tendermintUrl);
+
+          const response = await client.ibc.unverified.client.stateTm(ibcTest.clientId);
+          expect(response.chainId).toEqual("ibc-1");
+          // TODO: Fill these expectations out
```

Comment:
This result has a lot of properties. Do we need to assert against all of them?

Reply:
I think it is fine like this. Good to have the test in place. When someone is working on this particular call or we run into issues, we can fill out the test body more and more.